### DISCUSSION TITLE
Map os como SAAS

### DISCUSSION THREAD

**Discussion by marlontomaz** [RamonSilva20/mapos#2373] [Ideas]
Olá, boa tarde! 
Estou pensando em transformar o mapOS em um SAAS, e gostaria de saber se alguem já fez  essa alteração ? 

Pensei em incluir em cada tabela, uma coluna tenant_id,  e cadastrar o cnpj da empresa com o ID em cada usuario da mesma empresa.

Exemplo do sql - >
ALTER TABLE clientes ADD COLUMN tenant_id INT;

a consulta sql ficaria assim _ > 

SELECT * FROM clientes WHERE tenant_id = 470000000000000;    ( cnpj de exemplo ) 

tabela de inclusão 

INSERT INTO clientes (nome, email, tenant_id) VALUES ('João Silva', 'joao@example.com', 470000000000000);

**Comment by Pr3d4dor** [comment]
Precisaria mexer em vários outros detalhes além desse. 

No momento não temos interesse nessa feature pois o mapos está sendo refeito em Laravel e esse mapos feito em code igniter apenas irá receber correções no momento.
